module/signature/errors.go

```diff
@@ -111,7 +111,7 @@ func IsInsufficientSignaturesError(err error) bool {
 
 // IsDecodeSignerIndicesError returns whether err is about decoding signer indices
 func IsDecodeSignerIndicesError(err error) bool {
```

Comment:
It is really important to me to avoid this pattern of "functions that universally classify errors into expected and unexpected". I have included detailed explanations for my reasoning into [PR 2517](https://github.com/onflow/flow-go/pull/2517). [PR 2517](https://github.com/onflow/flow-go/pull/2517) consolidates the error handling and removes the function `IsDecodeSignerIndicesError`.